src/articleView-components/Article.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+export function Article({ article }) {
+  // const [articleBody, setArticleBody]=useState('')
+
+  return (
+    <>
```

Comment:
Fragments aren't great as they don't semantically tell us what is going on in the page. Conveniently there is a tag called article...which it looks like this is 👀